lib/order_repository.rb

```diff
@@ -0,0 +1,42 @@
+require_relative './order'
+require_relative './database_connection'
+
+class OrderRepository
+  def all
+    result_set = DatabaseConnection.exec_params('SELECT id, customer_name, date, item_id FROM orders;', [])
```

Comment:
Maybe assigning the arguments passed to exec_params() as variables before this line (like you've done for find_id() in item_repository.rb as sql and sql_params) would be a bit more readable here